Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+## Coaching Guide
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
+
+## How do I know if I am coaching?
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
+
+It is important that you do not vote for a goal if you are coaching so check with the coach team lead each week you are level 3 or above to determine if you are on coach-duty or not.
+
+## What does coaching entail?
+
+Check out the [Technical Coach](../Game_Manual/Levels_and_Roles.md#technical-coach) outline in [Levels and Roles](../Game_Manual/Levels_and_Roles.md).
+
+### Okay, but what does my day actually look like?
```

Comment:
Oh nevermind, I didn't see the `join morning stand-ups` item lol.  Linear reading FTL. ☹️  I would expand on that with the above, and change the `Join` to `Hold` so that is sounds more like the responsibility of the coach to facilitate these.